src/store/middlewares/block.js

```diff
@@ -1,73 +1,65 @@
-import io from 'socket.io-client';
 import actionTypes from '../../constants/actions';
 import { networkStatusUpdated } from '../../actions/network';
-import { getAPIClient } from '../../utils/api/network';
 import { olderBlocksRetrieved, forgingTimesRetrieved } from '../../actions/blocks';
+import { blockSubscribe, blockUnsubscribe } from '../../utils/api/block';
 
 const intervalTime = 5000;
 let interval;
-let connection;
-let forcedClosing = false;
```

Comment:
These goal variables are loose. This is Redux middleware and has direct access to Redux store. If we store these values in `store.network`, we update them traceably and follow the reactive pattern upon which is the application is build.

Reply:
I know you implemented it similar to what it was, and it's work as it did in the past, but I think we can take one more step of improvement here.